_posts/2021-02-25-

```diff
@@ -0,0 +1,55 @@
+---
+title:  "Notes: February 2021 Foundation for Public Code community call"
+date: 2021-02-25
+author: elena, ryan 
+type: blogpost
+excerpt: Introducing the publiccode.net product requirements document
+category: community call
+---
+
+
+# Notes: February 2021 Foundation for Public Code community call 
+
+In this community call, we talked through our thinking about [publiccode.net](https://publiccode.net/) as product ([pull request #169](https://github.com/publiccodenet/publiccode.net/pull/169)), and demo'd [our wireframes](https://www.figma.com/file/Mcz4evZVI4rRwv3quXtKGC/Site-Map-Wireframe-exploration-Foundation-for-Public-Code?node-id=177%3A0).
+
+
+## Attendees
+
+Alba Roza
+Ben Cerveny
+Boris van Hoytema
+Claus Mullie
+Elena Findley-de Regt (chair)
+Eric Herman
+Felix Faassen
+Jan Ainali
+Laura Scheske
+Martin Dias d'Ullois
+Rasmus Frey
+Ryan Conlan (guest speaker)
+
+   
+## Short update from the Foundation for Public Code
+
+- [Signalen is in incubation](https://blog.publiccode.net/news/2021/02/04/signalen-is-in-incubation.html) to fully meet the Standard for Public Code!
```

Comment:
I am not sure here, but does the first exclamation mark imply we should have periods on the other bullet points? Right now it looks a bit odd to me.